deploy/kubernetes/overlays/dev/kustomization.yaml

```diff
@@ -1,5 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
 bases:
 - ../../base
 patches:
```

Comment:
shouldn't the dev overlay also have the `rbac_add_snapshotter.yaml`

Reply:
Rebased dev overlay on top of alpha overlay